### PR TITLE
Ag Grid major version update

### PR DESCRIPTION
**Pull request by vadimpiven**

### CODE REVIEW

**Comment by vadimpiven** [comment]
As of now - AgGridVue component is broken, created bug in Zendesk https://ag-grid.zendesk.com/hc/en-us/requests/35746

File: sdk/ui-vue/src/aggrid.ts [hunk 1]
```diff
@@ -1,35 +1,9 @@
...
-    CsvExportModule,
-    ClipboardModule,
-    RangeSelectionModule,
-    RichSelectModule,
-    MenuModule,
-    ExcelExportModule,
-    SideBarModule,
-    ColumnsToolPanelModule,
-  ]);
+  ModuleRegistry.registerModules([AllEnterpriseModule]);
```
Comment (vadimpiven) [inline sdk/ui-vue/src/aggrid.ts:6]: attention!

File: sdk/ui-vue/vite.config.ts [hunk 2]
```diff
@@ -17,20 +17,8 @@ export default defineConfig({
...
-        '@ag-grid-community/vue3',
-        '@ag-grid-community/csv-export',
-        '@ag-grid-enterprise/core',
-        '@ag-grid-enterprise/clipboard',
-        '@ag-grid-enterprise/range-selection',
-        '@ag-grid-enterprise/rich-select',
-        '@ag-grid-enterprise/menu',
-        '@ag-grid-enterprise/excel-export',
-        '@ag-grid-community/theming',
+        'ag-grid-enterprise',
```
Comment (vadimpiven) [inline sdk/ui-vue/vite.config.ts:20]: is it needed here?

Comment (gramkin) [reply sdk/ui-vue/vite.config.ts:20]: Yes, definitely

**Pull request merged**